new-system-tests/src/test/resources/sql/00_persistent_volume_claim.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
It is recommended to use persistent storage for PostgreSQL database in production, so it will be nice to test this configuration, but we can probably use in-memory database storage for testing here, because I do not expect different behavior of PostgreSQL when using in-memory storage.

Reply:
you are entirely correct that the recommendation is to use PVCs, that said, PVCs are used to persist data on Pod restarts.
In these tests, this shouldn't happen so writing to an ephemeral volume or to a persistent one should be irrelevant for the test scenario.

> because I do not expect different behavior of PostgreSQL when using in-memory storage.


agree